README.md

```diff
@@ -127,6 +126,7 @@ Config:
 * local: `Boolean` Build local HTML, CSS and JS files, in addition to demo HTML for the build service. Also runs a local server to help you test your demos.
```

Comment:
Should probably remove the mention of running the server now that runServer is there.  Perhaps `runServer` could imply `local`?